finesse/gui/led_icons.py

```diff
@@ -0,0 +1,85 @@
+"""Class for LED Icons."""
+from importlib import resources
+
+from PySide6.QtCore import QTimer
+from PySide6.QtGui import QImage, QPixmap
+from PySide6.QtWidgets import QLabel
+
+img_files = resources.files("finesse.gui.images")
+poll_on_img_data = img_files.joinpath("poll_on.png").read_bytes()
+poll_off_img_data = img_files.joinpath("poll_off.png").read_bytes()
+alarm_on_img_data = img_files.joinpath("alarm_on.png").read_bytes()
+alarm_off_img_data = img_files.joinpath("alarm_off.png").read_bytes()
+
+poll_on_img = QImage.fromData(poll_on_img_data)
+poll_off_img = QImage.fromData(poll_off_img_data)
+alarm_on_img = QImage.fromData(alarm_on_img_data)
+alarm_off_img = QImage.fromData(alarm_off_img_data)
```

Comment:
All these variables are public so they'll be exported from the module. I'd make them private (by putting an underscore in front).